cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
```

Comment:
Defining payment as optional constant (let) makes it always nil. It should be var to allow setting from outside the class, or assigning it injected in initializer.